main.py

```diff
@@ -1,22 +1,40 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
-app = Flask(__name__)
-FlaskJSON(app)
+def create_app():
+    app = Flask(__name__)
+    return app
 
-@app.route("/")
-def hello():
-    return "Hello World!"
+# if __name__ == '__main__':
+app = create_app()
+app.run()
 
 @app.route('/login', methods=['POST'])
 def login():
+    try:
```

Comment:
It's a good idea to limit the scope of `try-except` to only lines which can throw the exception, namely the `json.loads` line.

Reply:
Done!